modules/FIC_Github.py

```diff
@@ -26,6 +28,33 @@ def _init_github(self, *args):
         self.repo_data = github3.GitHub.repository(args[0], args[1], args[2])
         return self.repo_data
 
+    def limit_checker(self):
+        limit_requests = self._gh.ratelimit_remaining
+        self.LOGGER.info("Rate limit is: %s", limit_requests)
```

Comment:
I don't think it's a good idea to log each time we check the limit, as the log filesize will get out of control.

Reply:
Fixed in commit https://github.com/mozilla-releng/firefox-infra-changelog/pull/416/commits/49c240f04cc66a18d794461d16e31bd08b2403fb
Thank you !